python/plugins/processing/algs/qgis/VoronoiPolygons.py

```diff
@@ -72,11 +72,15 @@ def __init__(self):
         super().__init__()
 
     def initAlgorithm(self, config=None):
-        self.addParameter(QgsProcessingParameterFeatureSource(self.INPUT, self.tr('Input layer'), [QgsProcessing.TypeVectorPoint]))
-        self.addParameter(QgsProcessingParameterDistance(self.BUFFER, self.tr('Buffer region'), parentParameterName=self.INPUT,
-                                                         minValue=0.0, maxValue=9999999999, defaultValue=0.0))
+        self.addParameter(QgsProcessingParameterFeatureSource(
+            self.INPUT, self.tr('Input layer'), [QgsProcessing.TypeVectorPoint]))
+        self.addParameter(
+            QgsProcessingParameterNumber(
+                self.BUFFER, self.tr('Buffer region (% of extent)'), parentParameterName=self.INPUT,
```

Comment:
Argh, missed this earlier, but you need to remove the parentParameterName argument. That doesn't exist for number parameters.

Reply:
Very sorry about that - I should have checked.  I have now removed it.